site/app/views/admin/PlagiarismView.php

```diff
@@ -8,72 +8,8 @@
 
 class PlagiarismView extends AbstractView {
 
-    public function plagiarismMainPage($all_configurations, $refresh_page, $nightly_rerun_info) {
+    public function plagiarismMainPage($plagiarism_result_info, $refresh_page) {
```

Comment:
Add types to function arguments since you are already editing it.

Reply:
Might as well add types to all of the functions in this file as well.  We didn't do it in the last refactor because we were waiting to do it in this PR.